### ISSUE TITLE
[Submission form] Field set and legend associations

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-evaluation-tools-list#27]
Under "Tool Functionality" please review the fieldset legend combinations. As you tab through the different elements in that section, JAWS and NVDA do not announce the legend content.

Maybe there is the one or the other id mixed up.
